generated/provider_dependencies.json

```diff
@@ -280,8 +280,8 @@
       "asgiref>=3.5.2",
       "cryptography>=2.0.0",
       "google-re2>=1.0",
-      "kubernetes>=21.7.0,<24",
-      "kubernetes_asyncio>=18.20.1,<25"
+      "kubernetes>=23.6.0,<29",
```

Comment:
23.6.0 is breaking with airflow 2.8.1. We need to bump the minimum Kubernetes version to be  28.1.0.
#37008. [Bug fix](https://github.com/kubernetes-client/python/pull/2095)